consul/consul.go

```diff
@@ -359,12 +361,11 @@ func marathonToConsulCheck(task *apps.Task, check apps.HealthCheck, serviceAddre
 			parsedURL.Host = fmt.Sprintf("%s:%d", serviceAddress, port)
 			consulCheck.HTTP = parsedURL.String()
 			return consulCheck
-		} else {
-			log.WithError(err).
-				WithField("Id", task.AppID.String()).
-				WithField("Address", serviceAddress).
-				Warnf("Could not parse provided path: %s", path)
 		}
+		log.WithError(err).
```

Comment:
Missing else?

Reply:
No, the return in the if block makes it unnecessary, the inter complained about this.